orchestra/orchestra_api.py

```diff
@@ -77,9 +77,10 @@ def create_orchestra_project(client,
     return project_id
```

Comment:
high-level comment across the PR: the API should just take `project_ids` to avoid redundancies. it will require a small refactor to anyone who calls `get_project_information`, but it will result in less logic duplication.

Reply:
Note that when you do this, you should clean up a few references as well:

```
docs/source/getting_started_transcript.rst:  get_project_information(project_id)
docs/source/getting_started_transcript.rst:  get_project_information(project_id)
journalism_workflow/management/commands/journalism_workflow_ctl.py:from orchestra.orchestra_api import get_project_information
journalism_workflow/management/commands/journalism_workflow_ctl.py:        p_info = get_project_information(project_id)
orchestra/interface_api/project_management/project_management.py:from orchestra.project_api.api import get_project_information
orchestra/interface_api/project_management/project_management.py:    project_information = get_project_information(project.id)
orchestra/orchestra_api.py:def get_project_information(project_id):
orchestra/project_api/api.py:def get_project_information(project_id):
orchestra/project_api/views.py:from orchestra.project_api.api import get_project_information
orchestra/project_api/views.py:        return get_project_information(project_id)
simple_workflow/management/commands/interactive_simple_workflow_demo.py:from orchestra.orchestra_api import get_project_information
simple_workflow/management/commands/interactive_simple_workflow_demo.py:        print('get_project_information(project_id)')
simple_workflow/management/commands/interactive_simple_workflow_demo.py:        project_info = get_project_information(project_id)
simple_workflow/management/commands/interactive_simple_workflow_demo.py:        project_info = get_project_information(project_id)
simple_workflow/management/commands/interactive_simple_workflow_demo.py:        print('get_project_information(project_id)')
```